### PR TITLE
test: delete iFrame function

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- remove experimental folder
- remove noIFrame from steps
- delete iFrame function
- delete type function override

### Current behaviour

As we no longer have an `experimental` folder we need to remove all files/folders in `cypress`.
As we no longer use an `iFrame` function we need to remove it and update all steps.

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

**Review by mmaciorowski-sage — APPROVED** [review]

File: cypress/features/regression/link.feature [hunk 1]
```diff
@@ -22,31 +22,31 @@ Feature: Link component
...
   @positive
   Scenario: Link is tabbable
-    When I open default "Link Test" component in noIFrame with "link" json from "commonComponents" using "tabbable" object name
+    When I open default "Link Test" component with "link" json from "commonComponents" using "tabbable" object name
     Then Link is tabbable
 
   @positive
   Scenario: Link is not tabbable
-    When I open default "Link Test" component in noIFrame with "link" json from "commonComponents" using "tabbableFlase" object name
+    When I open default "Link Test" component with "link" json from "commonComponents" using "tabbableFlase" object name
```
Comment (ZhuoyuJin) [inline cypress/features/regression/link.feature:30]: Should it be a spelling mistake for "tabbableFalse"?

Comment (DlgSHi) [reply cypress/features/regression/link.feature]: updated

**Review by ZhuoyuJin — APPROVED** [review]

File: cypress/features/regression/groupedCharacter.feature [hunk 2]
```diff
@@ -3,7 +3,7 @@ Feature: GroupedCharacter component
 
   @positive
   Scenario Outline: Set groups to <nameOfObject> and verify input
-    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
+    Given I open default "GroupedCharacter Test" component with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
     When I put "<example>" example grouped character in no Iframe
```
Comment (sianford) [inline cypress/features/regression/groupedCharacter.feature:7]: Why is "no Iframe" left in here?

File: cypress/features/regression/groupedCharacter.feature [hunk 3]
```diff
@@ -14,7 +14,7 @@ Feature: GroupedCharacter component
 
   @positive
   Scenario Outline: Set separator to <separator>
-    Given I open default "GroupedCharacter Test" component in noIFrame with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
+    Given I open default "GroupedCharacter Test" component with "groupedCharacter" json from "commonComponents" using "<nameOfObject>" object name
     When I put "<text>" example grouped character in no Iframe
```
Comment (sianford) [inline cypress/features/regression/groupedCharacter.feature:18]: Why is "no Iframe" left in here?

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature [hunk 4]
```diff
@@ -158,13 +158,13 @@ Feature: Accessibility tests - Design System folder
 
   @accessibility
   Scenario: Design System Select component default story page
-    Given I open "Design System Select" component page "default story" in no iframe
+    Given I open "Design System Select" component page "default story"
     When I click on Select input in noIframe
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityDesignSystem.feature:162]: Why is "noIframe" left in here?

File: cypress/features/regression/designSystem/filterableSelect.feature [hunk 5]
```diff
@@ -23,60 +23,60 @@ Feature: Design System Filterable Select component
...
+    Given I open "Design System Select filterable" component page "controlled"
     When I click on dropdown button
     Then "filterable" Select list is opened
 
   @positive
   Scenario: Close Filterable Select list by clicking out of component
-    Given I open "Design System Select filterable" component page "controlled" in no iframe
+    Given I open "Design System Select filterable" component page "controlled"
       And I click on dropdown button
     When I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/filterableSelect.feature:46]: Why is "no Iframe" left in here?

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature [hunk 6]
```diff
@@ -92,23 +92,23 @@ Feature: Accessibility tests - Design System folder
...
   Scenario: Design System Numeral Date component validations page
-    When I open "Design System Numeral Date Test" component page "validations" in no iframe
+    When I open "Design System Numeral Date Test" component page "validations"
     Then "Numeral Date" component has no accessibility violations
 
   @accessibility
   Scenario: Design System Popover Container component default page
-    Given I open "Design System Popover Container" component page "default story" in no iframe
+    Given I open "Design System Popover Container" component page "default story"
     When I open popover container in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityDesignSystem.feature:106]: Why is "NoIFrame" left in here?

File: cypress/features/regression/alert.feature [hunk 7]
```diff
@@ -30,13 +30,13 @@ Feature: Alert component
 
   @positive
   Scenario: Disable escape key
-    When I open default "Alert Test" component in noIFrame with "alert" json from "commonComponents" using "disableEscKey" object name
+    When I open default "Alert Test" component with "alert" json from "commonComponents" using "disableEscKey" object name
       And I press ESC onto focused element
     Then Alert is visible in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/alert.feature:35]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/accordion.feature [hunk 8]
```diff
@@ -3,24 +3,24 @@ Feature: Design System Accordion component
 
   @positive
   Scenario: I expand accordion using click
-    Given I open "Design System Accordion" component page "default story" in no iframe
+    Given I open "Design System Accordion" component page "default story"
     When I expand Design System accordionRow via click in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/accordion.feature:7]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/dateInput.feature [hunk 9]
```diff
@@ -21,7 +21,7 @@ Feature: Design System Date Input component
 
   @positive
   Scenario Outline: Change Date Input component label align to <labelAlign>
-    When I open default "Design System Date Input Test" component in noIFrame with "dateInput" json from "designSystem" using "<nameOfObject>" object name
+    When I open default "Design System Date Input Test" component with "dateInput" json from "designSystem" using "<nameOfObject>" object name
     Then label Align on preview is "<labelAlign>" in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/dateInput.feature:25]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/menu.feature [hunk 10]
```diff
@@ -71,7 +71,7 @@ Feature: Design Systems Menu component
 
   @positive
   Scenario: Check that the Search component is focusable by using the downarrow key
-    Given I open "Design System Menu" component page "submenu with search" in no iframe
+    Given I open "Design System Menu" component page "submenu with search"
       And I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/menu.feature:75]: Why is "no IFrame" left in here?

Comment (sianford) [reply cypress/features/regression/designSystem/menu.feature:75]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/menu.feature [hunk 11]
```diff
@@ -82,7 +82,7 @@ Feature: Design Systems Menu component
 
   @positive
   Scenario: Check that the Search component is focusable by using the uparrow key
-    Given I open "Design System Menu" component page "submenu with search" in no iframe
+    Given I open "Design System Menu" component page "submenu with search"
       And I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/menu.feature:86]: Why is "no iFrame" left in here?

File: cypress/features/regression/designSystem/menu.feature [hunk 12]
```diff
@@ -95,7 +95,7 @@ Feature: Design Systems Menu component
 
   @positive
   Scenario: Check that the Search component close icon is focusable in Menu when using keyboard navigation
-    Given I open "Design System Menu" component page "submenu with search" in no iframe
+    Given I open "Design System Menu" component page "submenu with search"
       And I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/menu.feature:99]: Why is "no iFrame" left in here?

File: cypress/features/regression/designSystem/multiSelect.feature [hunk 13]
```diff
@@ -20,53 +20,53 @@ Feature: Design System Multi Select component
...
       And I click on dropdown button
     When I click on dropdown button
     Then multi Select list is closed
 
   @positive
   Scenario: Close Multi Select list by clicking out of component
-    Given I open "Design System Select multiselect" component page "controlled" in no iframe
+    Given I open "Design System Select multiselect" component page "controlled"
       And I click on dropdown button
     When I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/multiSelect.feature:44]: Why is "no iFrame" left in here?

File: cypress/features/regression/designSystem/popoverContainer.feature [hunk 14]
```diff
@@ -3,20 +3,20 @@ Feature: Design System Popover container component
 
   @positive
   Scenario: Popover container is opened
-    Given I open "Design System Popover container" component page "default story" in no iframe
+    Given I open "Design System Popover container" component page "default story"
     When I open popover container in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/popoverContainer.feature:7]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/popoverContainer.feature [hunk 15]
```diff
@@ -3,20 +3,20 @@ Feature: Design System Popover container component
...
     Then Popover container is visible
 
   @positive
   Scenario: Popover container is closed
-    Given I open "Design System Popover container" component page "default story" in no iframe
+    Given I open "Design System Popover container" component page "default story"
       And I open popover container in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/popoverContainer.feature:13]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/simpleSelect.feature [hunk 16]
```diff
@@ -13,35 +13,35 @@ Feature: Design System Select component
...
       And I focus select input
     When I press Tab onto focused element
     Then "simple" Select list is closed
 
   @positive
   Scenario: Close Select list by clicking out of component
-    Given I open "Design System Select" component page "controlled" in no iframe
+    Given I open "Design System Select" component page "controlled"
       And I click on dropdown button
     When I click onto root in Test directory in no iFrame
```
Comment (sianford) [inline cypress/features/regression/designSystem/simpleSelect.feature:31]: Why is "no iFrame" left in here?

File: cypress/features/regression/designSystem/simpleSelect.feature [hunk 17]
```diff
@@ -80,28 +80,28 @@ Feature: Design System Select component
...
     Then "simple" Select list is opened
       And Option list has multiColumns header
       And Option list has multiColumns body
 
   @positive
   Scenario Outline: Show Select list is at the <position> in <size> viewport
-    Given I open Test default "Select" component in noIFrame with "simpleSelect" json from "test" using "<size>" object name
+    Given I open Test default "Select" component with "simpleSelect" json from "test" using "<size>" object name
       And I have a <size> viewport
     When I click on Select input in noIframe
```
Comment (sianford) [inline cypress/features/regression/designSystem/simpleSelect.feature:106]: Why is "noIframe" left in here?

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature [hunk 18]
```diff
@@ -12,26 +12,26 @@ Feature: Accessibility tests - Design System folder
 
   @accessibility
   Scenario: Design System Accordion component primary page opened state
-    Given I open "Design System Accordion" component page "default story" in no iframe
+    Given I open "Design System Accordion" component page "default story"
     When I expand Design System accordionRow via click in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/accessibility/accessibilityDesignSystem.feature:16]: Why is "NoIFrame" left in here?

File: cypress/features/regression/designSystem/filterableSelect.feature [hunk 19]
```diff
@@ -86,7 +86,7 @@ Feature: Design System Filterable Select component
 
   @positive
   Scenario: Full list options is visible when value is set and select is opened again
-    Given I open "Design System Select filterable" component page "default story" in no iframe
+    Given I open "Design System Select filterable" component page "default story"
       And I type "A" into simple select input in noIframe
```
Comment (sianford) [inline cypress/features/regression/designSystem/filterableSelect.feature:90]: Why is "noIframe" left in here?

File: cypress/features/regression/test/popoverContainer.feature [hunk 20]
```diff
@@ -3,7 +3,7 @@ Feature: Popover container component
 
   @positive
   Scenario Outline: Change Popover container component title to <title>
-    Given I open Test default "Popover container" component in noIFrame with "popoverContainer" json from "test" using "<nameOfObject>" object name
+    Given I open Test default "Popover container" component with "popoverContainer" json from "test" using "<nameOfObject>" object name
     When I open popover container in NoIFrame
```
Comment (sianford) [inline cypress/features/regression/test/popoverContainer.feature:7]: Why is "NoIFrame" left in here?

**Review by ZhuoyuJin — APPROVED** [review]

File: cypress/features/regression/test/loader.feature [hunk 21]
```diff
@@ -12,8 +12,8 @@ Feature: Loader default component
       | large   | 20     | 8      | sizeLarge    |
 
   @positive
-  Scenario Outline: Verify size of button with <size> loader
```
Comment (sianford) [inline cypress/features/regression/test/loader.feature:15]: Size seems to have been removed from here but it then won't specify the size when each example runs.

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by sianford — APPROVED** [review]

**Review by ZhuoyuJin — APPROVED** [review]

**Pull request merged**